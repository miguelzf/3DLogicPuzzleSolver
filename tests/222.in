- - - - - -
- - - - - -
- - - - - R
- - - B - -
- - - - - -
- - - - - L

- - L - - -
P - - - G -
- - - - - -
- C - - Y R
- - B - - -
- - - - - -

- - - - - -
- - - Y - -
- C - - - -
- - - - - -
- P - G - -
- - - - - -
