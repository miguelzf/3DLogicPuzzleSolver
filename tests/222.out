
Solution:
p  p  p  p  p  p  
p  c  c  c  c  c  
p  c  l  l  l  R  
p  c  l  B  l  -  
p  c  l  b  l  -  
p  c  l  b  l  L  
================
p  c  L  b  -  r  ||  r  r  r  r  c  p  
P  c  b  b  G  r  ||  y  y  y  Y  c  p  
-  c  b  g  g  r  ||  y  C  c  c  c  p  
-  C  b  g  Y  R  ||  y  p  p  p  p  p  
-  -  B  g  y  y  ||  y  P  -  G  -  -  
-  -  -  g  g  g  ||  g  g  g  g  -  -  
